ducktape/src/main/scala/io/github/arainko/ducktape/ArgBuilderConfig.scala

```diff
@@ -40,4 +40,12 @@ object Arg {
     ev2: FieldType <:< ArgType
   ): ArgBuilderConfig[Source, Dest, ArgSelector] = throw NotQuotedException("Arg.renamed")
 
+  @compileTimeOnly("'Arg.default' needs to be erased from the AST with a macro.")
```

Comment:
I don't think we should support that for `Arg` builders, you probably CAN get the default from the passed in method but just thinking about it makes me wince - let's drop this